pgjdbc/src/test/java/org/postgresql/test/jdbc3/Jdbc3BlobTest.java

```diff
@@ -278,7 +278,7 @@ public void readWriteStream(int offset, byte[] data) throws SQLException, IOExce
 
   @Test
   public void testPattern() throws SQLException {
-    byte[] data = "abcdefghijklmnopqrstuvwxyx0123456789".getBytes();
+    byte[] data = "abcdefghijklmnopqrstuvwxyz0123456789".getBytes();
```

Comment:
This sort of thing happens way more often than I'd like, and pretty much every variation I run into has many hits in cs.github.com, but I'm still shocked by them when I see them.